config.json

```
@@ -12,6 +12,20 @@
   "version": 3,
   "online_editor": null,
   "test_pattern": "_test\\.dart$",
+  "files": {
+    "solution": [
+      "lib/%{snake_slug}.dart"
+    ],
+    "test": [
+      "test/%{snake_slug}_test.dart"
+    ],
+    "example": [
+      "lib/example.dart"
+    ],
+    "exemplar": [
+      "path/to/%{kebab_slug}.ext"
```

https://github.com/exercism/docs/pull/58 Says this is optional, so leave it as is or empty array?

---

The field is optional because you don't _have_ to set it, but I'd set it to:

```suggestion
      ".meta/exemplar.dart"
```

---

I'm fine with your suggestion.
I have a question though, the `exemplar.dart` doesn't exist for any practice exercise, does that cause any breaking effects?